DateUtilsTest. getWeekdaysInMonth[jvm] test failing

## Pre-submission checklist
- [X] I am submitting a bug report, not a feature request.
- [X] I have searched for similar issues, but did not find any matches.

## Description
During the execution of `DateUtilsTest. getWeekdaysInMonth[jvm]`, the test fails. The rough description of the test is:
```
java.lang.AssertionError: 
Expected: [<4>, <4>, <4>, <4>, <5>, <5>, <5>]
     but: was [<4>, <4>, <4>, <4>, <4>, <4>, <4>]
...
```

I checked this test, and indeed, Febraury 2018 has different set of weekdays (4 mon, 4 tues, 4 weds, 4 fris, 4 sat, 4 suns).
The problem is that `GregorianCalendar` returns Time Zone dependent date, and in my case it was returning a one day before actual date.

## Steps to reproduce
Run `./build.sh build debug` command

## System information
- Any (CI issue)

## Screenshots
Calendar 2018 February:
![image](https://github.com/iSoron/uhabits/assets/28221162/15ddb3aa-912e-426a-8a85-6f51fc4057b1)

Source of error:
![image](https://github.com/iSoron/uhabits/assets/28221162/ee7509ca-6b23-4c38-b8da-e32ae93b3cf1)

And the rest of calendar tests

--- a/uhabits-core/src/jvmTest/java/org/isoron/uhabits/core/utils/DateUtilsTest.kt
+++ b/uhabits-core/src/jvmTest/java/org/isoron/uhabits/core/utils/DateUtilsTest.kt
@@ -120,10 +120,16 @@ class DateUtilsTest : BaseUnitTest() {
 
     @Test
     fun getWeekdaysInMonth() {
-        val february = GregorianCalendar(2018, Calendar.FEBRUARY, 1)
-        val leapFebruary = GregorianCalendar(2020, Calendar.FEBRUARY, 1)
-        val month = GregorianCalendar(2020, Calendar.APRIL, 1)
-        val longMonth = GregorianCalendar(2020, Calendar.AUGUST, 1)
+        fun getCalendarUTC(year: Int, month: Int, dayOfMonth: Int): GregorianCalendar {
+            val gregCalendar = GregorianCalendar(year, month, dayOfMonth)
+            gregCalendar.timeZone = TimeZone.getTimeZone("UTC")
+            return gregCalendar
+        }
+
+        val february = getCalendarUTC(2018, Calendar.FEBRUARY, 1)
+        val leapFebruary = getCalendarUTC(2020, Calendar.FEBRUARY, 1)
+        val month = getCalendarUTC(2020, Calendar.APRIL, 1)
+        val longMonth = getCalendarUTC(2020, Calendar.AUGUST, 1)
 
         assertThat(
             arrayOf(4, 4, 4, 4, 4, 4, 4),

--- a/uhabits-core/src/jvmTest/java/org/isoron/uhabits/core/utils/DateUtilsTest.kt
+++ b/uhabits-core/src/jvmTest/java/org/isoron/uhabits/core/utils/DateUtilsTest.kt
@@ -120,11 +120,10 @@ class DateUtilsTest : BaseUnitTest() {
 
     @Test
     fun getWeekdaysInMonth() {
-        fun getCalendarUTC(year: Int, month: Int, dayOfMonth: Int): GregorianCalendar {
-            val gregCalendar = GregorianCalendar(year, month, dayOfMonth)
-            gregCalendar.timeZone = TimeZone.getTimeZone("UTC")
-            return gregCalendar
-        }
+        fun getCalendarUTC(year: Int, month: Int, dayOfMonth: Int) =
+            GregorianCalendar(year, month, dayOfMonth).apply {
+                timeZone = TimeZone.getTimeZone("UTC")
+            }
 
         val february = getCalendarUTC(2018, Calendar.FEBRUARY, 1)
         val leapFebruary = getCalendarUTC(2020, Calendar.FEBRUARY, 1)

